### DISCUSSION TITLE
Creating Discussions for Godot-OpenKCC Project

### DISCUSSION THREAD

**Discussion by nicholas-maltbie** [nicholas-maltbie/Godot-OpenKCC#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask any questions about the Godot-OpenKCC project you may have.
  * Share ideas you have for the project or feature requests. (These discussion threads can be converted to issues)
  * Bring up any bugs or issues you find in the project. (These discussion threads can be converted to issues)
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nicholas-maltbie** [comment]
Hello, going to start off by introducing myself.

I'm Nick Maltbie, I've been creating and maintaining the Godot-OpenKCC project as a learning and exercise for Godot and Unity. I work as a software engineer. Hopefully I'll be able to make more YouTube videos explaining how this project works and differs from the Unity version.

Thanks to @jasonswearingen for suggesting opening a discussion in issue #27, probably an easier way to communicate than having to open GitHub issues.

**Comment by ndcrd** [comment]
Hi! I am Niko and am just started making my own games in Godot. I was working as web developer in my past, but gamedev was my dream since I was a kid. Can't really flex much on my gamedev projects. I am writing this post to thank @nicholas-maltbie for making a KCC for Godot engine. I integrated it with my own character controller that was working along side state-machines. Main issue was saying goodbye to a stock Character3d node, but there is so much on top of RigidBody3d with KCC now! Nightmares with stairs are no more 😄
